iceoryx_posh/include/iceoryx_posh/internal/roudi/roudi.hpp

```diff
@@ -49,15 +49,35 @@ class RouDi
         DEFER_START
     };
 
+    struct RoudiStartupParameters
```

Comment:
at one day, this needs to go to `RouDiConfig` and also made available in the TOML config

Reply:
That's probably also valid for some other predefined values (e.g. discovery interval, etc.), isn't it?

Reply:
I would see this topic in a separate PR/issue. I created it here: https://github.com/eclipse/iceoryx/issues/353